src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -181,6 +181,9 @@ def __init__(self, *args, **kwargs):
         self.context = None
         self.d_log = None
 
+        # Default to 10 minute timeout
+        self.timeout = 300
```

Comment:
This will have unintended consequences in all functional python tests and actually removes a test's ability to define it's own timeout.  

The `TestWithoutServers` class inherits the apricot `Test` class and the `TestWithoutServers.__init__()` method first calls `Test.__init__()`.  `Test.__init__()` defines the code which assigns the `self.timeout` value used by avocado with the option to assign the value from the test yaml.  With `TestWithoutServers.__init__()` assigning a `self.timeout` value after the `Test.__init__()` call, it has overridden the value.  As almost all functional tests are base on one of these classes, or  `TestWithServers` which inherits `TestWithoutServers`,  this change means all functional tests will now use either a 300 or 330 seconds (`TestWithServers` adds 30 seconds) timeout.

In addition to this problem, a default 5 minute timeout per cart test is very large.  It will not help CI execution to have a default that large.

Reply:
Ah, I see - thank you.  https://github.com/daos-stack/daos/pull/3850/commits/c1cda2578125f2a9ec65feff3f9609d89ffda556